config/app.php

```diff
@@ -149,15 +150,6 @@
         /*
          * Package Service Providers...
          */
-        Laravel\Tinker\TinkerServiceProvider::class,
-
-        /*
-         * BotMan Service Providers...
-         */
-        BotMan\Tinker\TinkerServiceProvider::class,
-        App\Providers\BotMan\DriverServiceProvider::class,
```

Comment:
I dont think `App\Providers\BotMan\DriverServiceProvider::class,` should be removed here as its an application provider, not of the package.

Reply:
You are right, added a commit to fix this. Thanks!